lib/galaxy/tool_util/verify/asserts/_util.py

```diff
@@ -59,9 +60,24 @@ def _assert_presence_number(
     expected = "Expected" if not negate else "Did not expect"
     if n is None and min is None and max is None:
         assert (not negate) == check_presence_foo(output, text), presence_text.format(
-            expected=expected, output=output, text=text
+            expected=expected, output=_nice_output(output), text=text
         )
     try:
         _assert_number(count_foo(output, text), n, delta, min, max, negate, n_text, min_max_text)
     except AssertionError as e:
-        raise AssertionError(str(e).format(output=output, text=text))
+        raise AssertionError(str(e).format(output=_nice_output(output), text=text))
+
+
+def _nice_output(output: Optional[str]):
+    """
+    shorten output if to long
+    """
+    if not output:
+        return output
+    if len(output) < 60:
```

Comment:
Maybe make that something like 100KB / 1MB ? It is sometimes very useful to see, and terminals are pretty fast these days ...

Reply:
So my original point was that I wanted to inspect the Galaxy log output before the assertion output. I struggled to find this by scrolling for minutes and was never sure if I might have scrolled to far and was looking at the previous run of the same test. Maybe there is better ways to do this (searching in the terminal, redirection). 

My problem with increasing the output to 100K or even 1MB is that it wont be obvious that the user does not see the complete output but a shortened one. Then I would prefer to keep things as they are. 

But maybe there is a better way to mark the output as shortened then

```
prefix
SNIP
suffix
```

?